config/labels/analyzers/clang-tidy.json

```diff
@@ -762,6 +782,11 @@
       "profile:extreme",
       "severity:HIGH"
     ],
+    "bugprone-unchecked-string-to-number-conversion": [
```

Comment:
please add to sei cert guideline err34-c
https://clang.llvm.org/extra/clang-tidy/checks/bugprone/unchecked-string-to-number-conversion.html